src/components/Form.js

```diff
@@ -255,6 +255,10 @@ class Form extends React.Component {
                 this.state.inputValues[inputID] = defaultValue;
             }
 
+            if (child.props.shouldDiscardDraft) {
```

Comment:
I don't think `shouldDiscardDraft` is the correct name because we do not reset any draft values; instead, we force the form to set the input value from the `defaultValue` props.

Reply:
I have changed the name to `isValidValueSaved`. The prop is from the child component so I think we don't need to add the prop types for it.